### PR TITLE
Update test-kfp.yml workflow.

### PR DESCRIPTION
**Pull request by revit13**
## Why are these changes needed?

This PR updates test-kfp.yaml workflow as was done for test-kfp-transform.template

### CODE REVIEW

File: .github/workflows/test-kfp.yml [hunk 1]
```diff
@@ -86,41 +83,30 @@ jobs:
...
-                  chmod 700 /tmp/get_helm.sh
-                  HELM_INSTALL_DIR=/tmp/ /tmp/get_helm.sh -v v${HELM_VERSION} --no-sudo
-                  chmod 777 /tmp/helm
-                  curl -L https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /tmp/kubectl
-                  chmod 777 /tmp/kubectl
-                  curl https://dl.min.io/client/mc/release/linux-amd64/mc --create-dirs -o /tmp/mc
-                  chmod +x /tmp/mc
-                  export DEPLOY_KUBEFLOW=1
-                  make -C $K8S_SETUP_SCRIPTS setup
+                  KFP_BLACK_LIST=$(./scripts/check-workflows.sh -show-kfp-black-list)
```
Comment (daw3rd) [inline .github/workflows/test-kfp.yml]: Can you move the black list down to just before the while loop?  THis way I know that it is not being used by the next few lines (assuming that is the case).  Needs to be done in two places.

**Review by daw3rd — APPROVED** [review]

File: .github/workflows/test-kfp.yml (hunk 1 above)
Comment (revit13) [reply .github/workflows/test-kfp.yml]: Done. Thanks

File: .github/workflows/test-kfp.yml [hunk 2]
```diff
@@ -9,7 +9,6 @@ on:
         tags:
             - "*"
         paths:
-            - ".make.*"
```
Comment (daw3rd) [inline .github/workflows/test-kfp.yml]: kfp/Makefile includes .make.defaults, so this should be included.

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: .github/workflows/test-kfp.yml (hunk 2 above)
Comment (revit13) [reply .github/workflows/test-kfp.yml]: Done. Thanks

**Review by daw3rd — APPROVED** [review]

**Pull request merged**